src/Illuminate/Database/Eloquent/InvalidIdFormatException.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Illuminate\Database\Eloquent;
+
+use Illuminate\Support\Arr;
+
+/**
+ * @template TModel of \Illuminate\Database\Eloquent\Model
+ */
+class InvalidIdFormatException extends \RuntimeException
```

Comment:
Wouldn't it make sense to extend the exception from [`\Symfony\Component\HttpKernel\Exception\HttpException`](https://github.com/symfony/symfony/blob/7.3/src/Symfony/Component/HttpKernel/Exception/HttpException.php) then?
```suggestion
use Symfony\Component\HttpKernel\Exception\HttpException;

/**
 * @template TModel of \Illuminate\Database\Eloquent\Model
 */
class InvalidIdFormatException extends HttpException
```